CMakeLists.txt

```diff
@@ -265,6 +265,7 @@ if("${DDC_BUILD_KERNELS_SPLINES}")
         set(KokkosKernels_ENABLE_COMPONENT_BLAS ON)
         set(KokkosKernels_ENABLE_COMPONENT_BATCHED ON)
         set(KokkosKernels_ENABLE_COMPONENT_LAPACK OFF)
+        set(KokkosKernels_ENABLE_COMPONENT_SPARSE ON)
```

Comment:
Maybe this is overkill to ask user to compile all KokkosSparse just to get the CooMatrix class. I can use a custom struct if you want.